static/js/compose_height.js

```diff
@@ -0,0 +1,52 @@
+import autosize from "autosize";
+import $ from "jquery";
+
+let is_full_size = false; // true or false
```

Comment:
We can call this `full_size_status` if we need a new variable name as discussed below.

Reply:
Sure, changed it.